.circleci/test-deploy.yml

```diff
@@ -16,6 +16,10 @@ release-filters: &release-filters
   branches:
     ignore: /.*/
 
+filters_non_master_only: &filter_non_master_only
```

Comment:
🐼 

Maybe opportunistic, but good to align CircleCI YAML conventions for `filters_non_master_only` & `release-filters`!